### PR TITLE
[ADP-3224] Fix bash instructions to start node and wallet containers in README

### PR DESCRIPTION
**Pull request by paolino**
- [x] Fix broken docker instructions in README

ADP-3224

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +86,8 @@ fi
 
 # set the node tag and wallet tag to compatible versions
 export NODE_TAG=8.1.2
-export WALLET_TAG=v2023-12-18
+export WALLET_TAG=2023.12.18
```
Comment (Anviking) [inline README.md:89]: This `WALLET_TAG` is needed by docker right, and is not unused?

**Review by paweljakubas — APPROVED** [review]
:+1:

**Review by Anviking — APPROVED** [review]
Have not run the whole thing, but lgtm

File: README.md (hunk 1 above)
Comment (paolino) [reply README.md:89]: yes, it's needed in the docker-compose

**Pull request merged**